tracing-script/generate.py

```diff
@@ -148,6 +148,9 @@ def truncate_file(path):
 # Copy generated sources.
 subprocess.call(['cp objs/*.c objs/*.h "{}"'.format(outdir_objs)], shell=True)
 
+# use soft link to avoid multiple copies of the same sources
+subprocess.call(['ln -s {}/src/ "{}/src"'.format(os.getcwd(),outdir)], shell=True)
```

Comment:
Is this really needed?

As far as I can tell, this will simplify immediate compilation. But I don't understand why anyone will go through running this script just to compile in place. Isn't it easier to just use Nginx's own build system instead?

Reply:
Well, never mind. 

Actually, I don't just use CMake as a build system. I use it as a bridge to my favorite IDE , Eclipse CDT. 
The generated CDT projects by CMake works very well in Eclipse. 
Well, maybe it is better to replace the arguments of  '' CMake add_executable  command"  from relative paths to absolute paths.  

ps: I love your script. :+1: